cmake/Summary.cmake

```diff
@@ -48,6 +48,7 @@ function(caffe2_print_configuration_summary)
   message(STATUS "  BUILD_TEST            : ${BUILD_TEST}")
   message(STATUS "  BUILD_JNI             : ${BUILD_JNI}")
   message(STATUS "  BUILD_MOBILE_AUTOGRAD : ${BUILD_MOBILE_AUTOGRAD}")
+  message(STATUS "  CROSS_COMPILING_MACOSX : ${CROSS_COMPILING_MACOSX}")
```

Comment:
should we add this to `aten/src/ATen/Version.cpp` to indicate whether pytorch was built with cross compilation? (easier to check later when we only have the binary released)

Reply:
Do you mean editing the std::string show_config() function to mention this variable?

Reply:
yes :-)

Reply:
We can, but why does it matter? Native and cross compilation should yield the same results performancewise.

Reply:
I would also guard this status print with `if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")` since this setting is meaningless on other platforms, isn't it?

Reply:
i see. i was thinking of in terms of debugging purpose - user could report it when running collect_env.py